VMBackup/main/backuplogger.py

```diff
@@ -51,7 +53,7 @@ def enforce_local_flag(self, enforced_local):
     def log(self, msg, local=False, level='Info'):
         #self.hutil.log("Started backuplogger log function")
         if(self.enforced_local_flag_value == False and self.logging_off == True):
-            self.hutil.log("enforced_local_flag_value is False and logging_off is true")
+            #self.hutil.log("enforced_local_flag_value is False and logging_off is true")
             return
         WriteLog = self.hutil.get_strvalue_from_configfile('WriteLog','True')
         if (WriteLog == None or WriteLog == 'True'):
```

Comment:
Is any change required for line 72 self.hutil.log("WriteLog is neither None nor 'True'.")

Reply:
removed that as well to be safe.